[Security] Access Tokens (#16819)
symfony/symfony-docs: security.rst

You can learn all about this authenticator in :doc:`/security/login_link`.

Access Tokens

> ~~~~~~~~~~~~~~

Reviewer: ```suggestion
~~~~~~~~~~~~~
```

Access Tokens are often used in API contexts.

> The user will receive a token from an authorization server

Reviewer: ```suggestion
The user receives a token from an authorization server
```

> The user will receive a short-lived token from an authorization server

Reviewer 2: ```suggestion
The user will receive a token from an authorization server
```

this is dependent on the implementation isn't?

> which will authenticate them.

Reviewer: ```suggestion
which authenticates them.
```
